pkg/jwt/github/provider.go

```diff
@@ -2,10 +2,17 @@ package github
 
 import (
 	"net/http"
+	"sync"
 
 	"github.com/dgrijalva/jwt-go"
+	"github.com/google/go-github/github"
 	"github.com/hellofresh/janus/pkg/config"
 	"github.com/hellofresh/janus/pkg/jwt/provider"
+	"github.com/pkg/errors"
+)
+
+const (
+	janusOwners = "janus-owners"
```

Comment:
This should come from an environment variable `JANUS_ADMIN_TEAM` and should be passed to the global config